kernel/init.c

```diff
@@ -51,7 +51,7 @@ const char * const build_timestamp = BUILD_TIMESTAMP;
 /* boot time measurement items */
 
 #ifdef CONFIG_BOOT_TIME_MEASUREMENT
-u64_t __noinit __start_time_stamp; /* timestamp when kernel starts */
+u64_t __start_time_stamp; /* timestamp when kernel starts */
```

Comment:
this now causes the number to always be zero? It is an important KPI to track on many platforms, i.e. to measure the time it takes to bootup zephyr including the bootloader. Right now it shows on x86 platforms:

tc_start() - Boot Time Measurement
Boot Result: Clock Frequency: 32 MHz
__start       : 7001108 cycles, 218784 us
_start->main(): 8489 cycles, 265 us
_start->task  : 8526 cycles, 266 us
_start->idle  : 328272 cycles, 10258 us
Boot Time Measurement finished

with the patch:

tc_start() - Boot Time Measurement
Boot Result: Clock Frequency: 32 MHz
__start       : 0 cycles, 0 us
_start->main(): 26453 cycles, 826 us
_start->task  : 26490 cycles, 827 us
_start->idle  : 346236 cycles, 10819 us
Boot Time Measurement finished